### ISSUE TITLE
Adding custom library from git fails due to missing import

### ISSUE THREAD

**Issue by chovanecadam** [DannyBen/bashly#440]
### Bashly Version

Latest Ruby Gem

### Description

[/lib/bashly/library_source.rb](https://github.com/DannyBen/bashly/blob/7cf3ef08babcb38c5dca860810660b00dbc01a35/lib/bashly/library_source.rb#L50) calls `Dir.mktmpdir 'bashly-libs-'` without `require 'tmpdir'`. This causes the call to fail. Adding the `require` to the top of the file fixed the issue for me.

### Reproduction Steps

Initialize a new Bashly project and then call the following (or any other git library)

```bash
bashly add --source git:https://gitlab.com/staticnoise/bashlib.git all
```

The call will fail with the following output:

```
Performing git operations, this may take a while...

NoMethodError
undefined method `mktmpdir' for Dir:Class
Did you mean?  mkdir
```

**Comment by DannyBen** [comment]
Thanks Adam. You are absolutely right.
When the tests run, they probably already load this `tmpdir` library prior to running the failing code - this is why this bug is not caught in tests.

**Comment by DannyBen** [comment]
This is now fixed and released in v1.1.1.
Thanks again.

**Comment by chovanecadam** [comment]
Happy to help.
